source/_integrations/openweathermap.markdown

```diff
@@ -71,8 +71,8 @@ The Weather entity provides data only in English. Home Assistant automatically t
 | `humidity`               | Humidity, %.                                                                                                                      |
 | `precipitation_kind`     | The kind of precipitation (Rain, Snow, Snow and Rain, None) for the last hour.                                                    |
 | `pressure`               | Atmospheric pressure at sea level, hPa.                                                                                           |
-| `rain`                   | Rain volume for the last hour, mm.                                                                                                |
-| `snow`                   | Snow volume for the last hour, mm.                                                                                                |
+| `rain`                   | Average rain rate since the last update, mm/hr.                                                                                   |
+| `snow`                   | Average rain rate since the last update, mm/hr.                                                                                   |
```

Comment:
I'm not sure that it's correct to say average rain since the last update.  Does that mean since the integration polled for a new number, or since OpenWeatherMap got new data?  

It might be less ambiguous to use the wording they have on their [API](https://openweathermap.org/api/one-call-3) which has `precipitation, mm/h`.

Or maybe 
`rain precipitation, mm/h`.

`snow precipitation, mm/h`.